extra/zabbix/vpoller-module/vpoller.c

```diff
@@ -30,10 +30,11 @@
 #include <zmq.h>
 
 #include "threads.h"
-#include "cfg.h"
-#include "sysinc.h"
+#include "zbxcfg.h"
+#include "zbxsysinc.h"
 #include "module.h"
-#include "log.h"
+#include "zbxlog.h"
+#include "zbxstr.h"
 
 #define VPOLLER_MODULE_VERSION "0.5.7"
```

Comment:
Please bump the version of the module here. Since this PR targets a major version, please update it to `0.6.0` at least.

Reply:
Resolved in new commit :D